src/pages/settings/Profile/ProfilePage.js

```diff
@@ -190,17 +184,46 @@ class ProfilePage extends Component {
         }, true);
     }
 
-    validateInputs() {
-        const [hasFirstNameError, hasLastNameError, hasPronounError] = ValidationUtils.doesFailCharacterLimit(
-            50,
-            [this.state.firstName.trim(), this.state.lastName.trim(), this.state.pronouns.trim()],
-        );
-        this.setState({
-            hasFirstNameError,
-            hasLastNameError,
-            hasPronounError,
-        });
-        return !hasFirstNameError && !hasLastNameError && !hasPronounError;
+    validate(values) {
+        const errors = {};
+
+        if (values.firstName) {
+            const [hasFirstNameError] = ValidationUtils.doesFailCharacterLimit(
+                50,
```

Comment:
Let's add it to `CONST.js`

Reply:
I wonder what specific advantage we get by moving

```
const [hasFirstNameError] = ValidationUtils.doesFailCharacterLimit(
                50,
                [values.firstName.trim()],
            );
```
and other similar validation check to `CONST.js`. Wouldn't it just replace `ValidationUtils.doesFailCharacterLimit` with `CONST.doesFailCharacterLimit`? Additionally, having that function inside `ValidationUtils` makes more sense.

Thoughts?

Reply:
Sorry that I wasn't clear. I meant we should move the limit **`50`** to `CONST.js`

Reply:
Ah, makes sense! Did the refactoring and pushed the changes.